scripts/run_kickstart_tests.sh

```diff
@@ -305,8 +305,13 @@ if [[ "$TEST_REMOTES" != "" ]]; then
     # that qemu on the remote systems can read.
     for remote in ${TEST_REMOTES}; do
         ssh kstest@${remote} mkdir kickstart-tests
-        scp -r . kstest@${remote}:kickstart-tests/
-        scp ${IMAGE} kstest@${remote}:kickstart-tests/
+        ssh kstest@${remote} mkdir install_images
+        rsync -az --delete . kstest@${remote}:kickstart-tests/
+        rsync -az ${IMAGE} kstest@${remote}:install_images/
+        # remove any old images
+        ssh kstest@${remote} find install_images -type f ! -name ${_IMAGE} -delete
+        # copy the image to the kickstart-tests folder, so guestfs can use it
+        ssh kstest@${remote} install_images/${_IMAGE} kickstart-tests/
```

Comment:
Maybe I'm blind but aren't you missing remote command here?

Reply:
Or maybe there should be `scp` instead of `ssh`?

Reply:
That's a good point - that does not look correct. I wonder how it still all works like this. :D